src/sdk/glacier.js

```diff
@@ -185,20 +220,20 @@ class GlacierBackend {
 
             return {
                 ongoing: true,
-                state: GlacierBackend.RESTORE_STATUS_ONGOING,
+                state: Glacier.RESTORE_STATUS_ONGOING,
             };
         } else {
             if (!restore_expiry || restore_expiry <= now) {
                 return {
                     ongoing: false,
-                    state: GlacierBackend.RESTORE_STATUS_CAN_RESTORE,
+                    state: Glacier.RESTORE_STATUS_CAN_RESTORE,
                 };
             }
 
             return {
                 ongoing: false,
                 expiry_time: restore_expiry,
-                state: GlacierBackend.RESTORE_STATUS_RESTORED,
+                state: Glacier.RESTORE_STATUS_RESTORED,
```

Comment:
in this case, the object is not externally managed (meaning xattr.storage_class=GLACIER), but if config.USE_DMAPI is on, we can still look at the dmapi state to verify that indeed the file is premigrated, and print a warning.